doc/getting_started/getting_started.rst

```diff
@@ -211,6 +214,65 @@ without the Zephyr SDK`_ for details.
       :board: arduino_due
       :goals: build
 
+Make sure to unset the ZEPHYR_SDK_INSTALL_DIR if you don't use the
+SDK's host tools. See `Building without the Zephyr SDK`_ for details.
+
+It is possible to use the Zephyr SDK's host tools along with a 3rd
+party cross compiler. To do that, keep the ZEPHYR_SDK_INSTALL_DIR
+environment variable set to the Zephyr SDK installation directory.
+See `Set Up the Development Environment`_ for more details on the
+ZEPHYR_SDK_INSTALL_DIR environment variable.
+
+Using Custom Cross Compilers
+============================
+
+To use a custom cross compiler, follow the steps below.
+
+#. Install a cross compiler suitable for your system. We will use the
+   gcc-arm-none-eabi compiler on Debian system for this example.
+
+   .. code-block:: console
+
+      # On Debian or Ubuntu
+      sudo apt-get install gcc-arm-none-eabi
+      # On Fedora or Red hat
+      sudo dnf install arm-none-eabi-newlib
+
+#. Build the example :ref:`hello_world` project, enter:
+
+   .. code-block:: console
+
+      # On Linux/macOS
+      unset GCCARMEMB_TOOLCHAIN_PATH
+      export ZEPHYR_TOOLCHAIN_VARIANT=cross-compile
+      export CROSS_COMPILE=/usr/bin/arm-none-eabi-
+      # On Windows
+      set GCCARMEMB_TOOLCHAIN_PATH=
+      set ZEPHYR_TOOLCHAIN_VARIANT=cross-compile
+      set CROSS_COMPILE=/usr/bin/arm-none-eabi-
```

Comment:
I think you can omit the Windows example.

/usr/bin/arm-none-eabi-

isn't a valid windows path.

Reply:
"Or, would it be OK to delete about windows from the custom compiler section?"
Yes